app/gui2/src/components/GraphEditor/widgets/WidgetFunction.vue

```diff
@@ -114,6 +114,71 @@ const selfArgumentExternalId = computed<Opt<ExternalId>>(() => {
 const visualizationConfig = computed<Opt<NodeVisualizationConfiguration>>(() => {
   // Even if we inherit dynamic config in props.input.dynamicConfig, we should also read it for
   // the current call and then merge them.
+  function collectArgumentNamesAndUuids() {
```

Comment:
@farmaazon, can you please advice (point me to an example) how to write (unit) tests for this behavior?

Reply:
Basic unit test written in 59ada87 - I can do more, but it'd be good to know how to debug the tests?